src/pymor/models/loader.py

```diff
@@ -0,0 +1,116 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import yaml
+
+from pymor.models.basic import StationaryModel
+
+# from pymor.parameters.base import Parameters
+
+from pymor.tools.io.matrices import load_matrix
+
+"""Loads models from yml-based configuration files.
+
+The configuration file should be of form:
+
+type: <subclass of pymor.models>
```

Comment:
Would you say we should version our model spec?

Reply:
I'd say we can do that when issues with different versions arise.

Reply:
There are no different versions then though...

Reply:
That's true ..

Reply:
Do you see a downside then of adding an attribute like `spec-version` then?
My idea would for now would just be to assert the current version in loader code. If we break the spec in the future we'll add a link to the migration guide and that's about it. I just want to avoid "weird" user facing exceptions if we can. And this seems very easy to do here.